src/tqec/plaquette/plaquette.py

```diff
@@ -18,6 +23,7 @@ def __init__(
         self,
         qubits: PlaquetteQubits,
         circuit: ScheduledCircuit,
+        measurements: list[Measurement] | None = None,
```

Comment:
I think I had a use-case in mind when writing it that way that might be hard to solve with "deriving from the `circuit`" but I cannot find it back and I agree it would be better to have it automatically computed.